crates/sui-graphql-rpc/src/types/owner.rs

```diff
@@ -36,6 +39,9 @@ pub(crate) struct Owner {
 /// Type to implement GraphQL fields that are shared by all Owners.
 pub(crate) struct OwnerImpl {
     pub address: SuiAddress,
+    /// The version of the object to use when querying dynamic fields. If not set, defaults to using
+    /// the latest checkpoint.
+    pub version: Option<u64>,
```

Comment:
Therefore you also don't need this here, you just add the version parameter to all the dynamic field functions, and pass the version up from all the sub-types.